apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/behandling/Grunnlagmapper.kt

```diff
@@ -90,36 +92,37 @@ fun Grunnlag.mapVerge(
     adresseService: AdresseService,
 ): Verge? =
     with(this) {
-        val relevantVerge =
-            hentRelevantVerge(
-                soeker.hentSoekerPdlV1()!!.verdi.vergemaalEllerFremtidsfullmakt,
+        val verger =
+            hentVerger(
+                soeker.hentSoekerPdlV1()!!.verdi.vergemaalEllerFremtidsfullmakt ?: emptyList(),
                 soeker.hentFoedselsnummer()?.verdi,
             )
-        if (relevantVerge != null) {
-            val mottakerAdresse =
-                runBlocking {
-                    adresseService.hentMottakerAdresse(
-                        sakType,
-                        relevantVerge.vergeEllerFullmektig.motpartsPersonident!!.value,
-                    )
-                }
-            return Vergemaal(
-                mottakerAdresse.navn,
-                relevantVerge.vergeEllerFullmektig.motpartsPersonident!!,
+        return if (verger.size == 1) {
+            val vergeFnr = verger.first().vergeEllerFullmektig.motpartsPersonident!!
+            Vergemaal(
+                navnViaAdresse(adresseService, sakType, vergeFnr),
+                vergeFnr,
             )
-        }
-
-        if (sakType == SakType.BARNEPENSJON) {
-            if (erOver18(brevutfallDto)) {
-                null
-            } else {
-                hentForelderVerge()
-            }
+        } else if (verger.size > 1) {
+            UkjentVergemaal()
```

Comment:
Burde bruker fått litt info her? 
Og burde det vært logget litt for vår egen del?

Reply:
Ja, det er nok lurt